tools/packet_generator_provisioning.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+PROJECT_ROOT=${PROJECT_ROOT:-$(cd ../ ; pwd -P)}
+DEPLOY_NAME=${DEPLOY_NAME:-cnftestbed}
+NODE_FILE=${NODE_FILE:-$(pwd)/data/$DEPLOY_NAME/packet_gen.env}
+NIC_FILE=${NIC_FILE:-$(pwd)/data/$DEPLOY_NAME/packet_gen_nics.env}
+
+NIC_TYPE=${NIC_TYPE:--e quad_intel=true}
+FACILITY=${FACILITY:-sjc1}
+VLAN_SEGMENT=${VLAN_SEGMENT:-$DEPLOY_NAME}
+PLAYBOOK=${PLAYBOOK:-packet_generator.yml}
+
+if ! [ -z ${PKTGEN_HOSTS+x} ]; then
+    HOSTS="$PKTGEN_HOSTS,"
+    PKTGEN_IPS_ARRAY=($(echo $PKTGEN_HOSTS | tr ',' ' '))
+elif ! [ -z ${NODE_FILE+x} ]; then
+    PKTGEN_IPS_ARRAY=($(yq r $NODE_FILE nodes.[*].addr | tr -d '\n''-'))
```

Comment:
`yq` must be installed on the machine running the deployment

Reply:
This commit should resolve this: 
https://github.com/cncf/cnf-testbed/pull/320/commits/71be8552ca212f7d60ea4e6293c21a49b9c6d5cf